manifests/templates/otel-collector.yaml

```diff
@@ -152,10 +152,6 @@ spec:
         volumeMounts:
         - name: otel-collector-config-vol
           mountPath: /conf
-        livenessProbe:
```

Comment:
This one is the collector in monitoring namespace, we could keep the liveness probe here as a deadman switch? I just realized it's given 200m CPU, whereas the otel-agent that now uses same image as otel-collector is only given 10m as a container in reconciler.

Reply:
I don't think it's necessary here either. Like I said in chat, livenessProbe is really only recommended to be added to work around problems with apps you don't control that can get wedged and don't recover automatically.

Reply:
👌 sgtm.